src/Reflectivity/Breakpoint.class.st

```diff
@@ -288,14 +316,36 @@ Breakpoint >> enable [
 { #category : #api }
 Breakpoint >> enabled: aBoolean [
 	enabled := aBoolean
-	
+]
+
+{ #category : #accessing }
+Breakpoint >> group [
+	"returns the group of the breakpoint"
+	^ group.
+]
+
+{ #category : #api }
+Breakpoint >> group: aBreakpointGroup [
+	"puts the breakpoint in a new group and sets the group variable to that group"
+	group removeBreakpoint: self.
```

Comment:
as the group uses a Set, I think we do not need to remove before adding (as the set will just ignore if we add twice)

Reply:
this is to remove the breakpoint from its old group, so it is necessary. At the moment a breakpoint is only intended to be in one group at once (except for the "All Breakpoints" group).

Reply:
ah, ok